spec/models/programme_detail_spec.rb

```diff
@@ -0,0 +1,133 @@
+require "rails_helper"
+
+describe ProgrammeDetail do
+  let(:trainee) { build(:trainee) }
+
+  subject { described_class.new(trainee: trainee) }
+
+  describe "validations" do
+    it { is_expected.to validate_presence_of(:subject) }
+
+    describe "custom" do
+      translation_key_prefix = "activemodel.errors.models.programme_detail.attributes"
+      before do
```

Comment:
could we add some spacing in this spec, some scenarios feel a bit squished

Reply:
cool, made it less dense